Source/AnimationMaker.swift

```diff
@@ -60,7 +60,7 @@ public final class AnimationMaker {
         return animations
     }
 
-    /// TODO: need to change property after animation completion
+    // TODO: need to change property after animation completion
     /// to make animation sequences relatively to each other
```

Comment:
This seems weird, so what we will do? Use double-slash comments or disable swiftformat for this rule?

Reply:
Cant we configure it? I think we can, need to check it though

Reply:
Unfortunately it happens due to `todos` rule, but we can't configure it in detail, so double-slash is preferred for `// TODO` and `// MARK`